components/tabs/tabs.js

```diff
@@ -279,7 +278,14 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(RtlMixin(FocusVisiblePolyf
 		};
 
 		this.arrowKeysOnBeforeFocus = async(tab) => {
+			const prevFocusable = this._tabInfos.find(info => info.activeFocusable);
+			if (prevFocusable) prevFocusable.activeFocusable = false;
+
 			const tabInfo = this._getTabInfo(tab.controlsPanel);
+			tabInfo.activeFocusable = true;
+
+			this.requestUpdate();
```

Comment:
Are you sure we don't need to `await this.updateComplete`?  Otherwise the active focusable tabindex values might not be updated by the time the arrow keys mixin calls `focus`...?  Or are we effectively relying on the code below to provide that delay?  If it doesn't need to be scrolled, and it doesn't need to be expanded, then there might not be any other delay.  This is something to check if you haven't already.

Reply:
I think you're probably right. In practice I never ran into this scenario and I sort of just mimicked what the code does when a new tab is selected, but I imagine there's some scenario where the timing wouldn't work as expected.